### PR TITLE
Add initial share logs page

### PR DESCRIPTION
**Pull request by mlejva**
### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes. 
- [x] I have run the following commands against my code to ensure it passes our linters:
    ```shell
    black .
    isort .
    mypy
    autoflake --remove-all-unused-imports --recursive --ignore-init-module-imports --ignore-pass-after-docstring autogpt tests --in-place
    ```

No need to add tests. I just added content to docs.

### CODE REVIEW

File: docs/share_logs.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## Share your logs with us to help improve Auto-GPT
+
+Do you notice weird behavior with your agent? Do you have an interesting use case? Do you have a bug you want to report?
+Follow the steps below to enable your logs and upload them. You can include these logs when making an issue report or discussing an issue with us.
+
+### Enable Debug Logs
+Activity and error logs are located in the `./output/logs`
```
Comment (Pwuts) [inline docs/share_logs.md]: This is incorrect, same as the current usage documentation :x

File: mkdocs.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@ nav:
   - Contributing:
     - Contribution guide: contributing.md
     - Running tests: testing.md
+    - Share your logs with us: share_logs.md
```
Comment (Pwuts) [inline mkdocs.yml]: Should be the top entry imo :)

File: docs/share_logs.md (hunk 1 above)
Comment (mlejva) [reply docs/share_logs.md]: @NeonN3mesis can you please take a look here to fix the issue @Pwuts is hinting at? I'd be happy to fix it myself but I'm not sure what part is outdated exactly

Comment (NeonN3mesis) [reply docs/share_logs.md]: I'm on it

Comment (NeonN3mesis) [reply docs/share_logs.md]: I made a new PR on your branch @mlejva

Comment (mlejva) [reply docs/share_logs.md]: I merged it

**Comment by mlejva** [comment]
I created a new PR #4965 with all the fixed changes. @Pwuts can you please take a look once you have a moment?

**Pull request closed without merging**